data/index.php

```diff
@@ -15,6 +15,12 @@
 		}, urldecode($_GET['ids']));
 	}
 	else $ids = false;
+
+	//Option for limiting the number of results, making sure it is numeric to prevent injection.
+	//Perhaps this should allow for A,B to allow paging?
+	if(isset($_GET['limit']) && is_numeric($_GET['limit']) && $_GET['limit']>0){
+		$LIMIT = ' LIMIT = '+$_GET['limit'];
```

Comment:
I think this should use . rather than + for concatenation?

Reply:
Indeed. It's been a lot of javascript lately, so my PHP-Fu is not what it used to be. :)